SADL/AM-CDM-testInspectionCharacterization.sadl

```diff
@@ -1,39 +1,23 @@
 uri "http://kdl.ge.com/AM-CDM-testInspectionCharacterization" alias AM-CDM-testInspectionCharacterization version "$Revision:$ Last modified on   $Date:$".
 
-import "http://kdl.ge.com/AM-CDM-build".
-
-TestInspectionCharacterization (note "Any test, inspection, or characterization performed on any part or material specimen from any stage of the additive manufacturing fabrication process (excluding in-process monitoring).") is a type of ProcessStep,
-	described by ticID (note "An identifier of the test, inspection or characterization type.") with a single value of type string,
-	described by ticName (note "A short description of the test, inspection, or characterization.") with a single value of type string,
-	described by ticType (note "Type of test/inspection/characterization, such as tensile test, fatigue test, etc.") with a single value of type TICType,
-	described by ticSpecimen with a single value of type Specimen,
-	described by ticLocation (note "The physical location where the test was conducted.") with a single value of type Address,
-	described by ticNotes (note "Itemized descriptions of observations relating to the individual test, inspection, or characterization of an individual specimen.") with values of type string,
-	described by ticSoftware (note "Identifier of any Software that was used during the test, inspection, or characterization.") with a single value of type Software,
-	described by ticDestructiveVsNonDestructive (note "Indication of whether the test, inspection, or characterization irreversibly changed the nature of the specimen.") with a single value of type TestType,
-	described by ticDuration (note "The time that the Test took (Test only).") with a single value of type Measurement,
-	described by ticSpecification (note "The specification to use to run the test, inspection, or characterization. Note that ticSpecification should override processSpecification") with a single value of type TICSpecification,
-	described by ticResult with a single value of type TestResult.
+import "http://kdl.ge.com/AM-CDM-Base".
+import "http://kdl.ge.com/AM-CDM-manufacturingProcess".
 
-TICSpecification (note "A specification with the details necessary to perform a test, specification, or characterization.") is a type of ProcessSpecification,
-	described by ticProcedure (note "The procedure used if it is not from an existing standard.") with a single value of type Document.
+TestInspectionCharacterization (note "Any test, inspection, or characterization performed on any part or material specimen from any stage of the additive manufacturing fabrication process (excluding in-process monitoring).") is a type of ManufacturingProcessStep,

```

Comment:
I agree that test is semantically different than a manufacturing process. However, I think it has identical properties. Is this a case where the name of ManufacturingProcessStep simply needs to be generalized?

For reference, with inheritance, a ManufacturingProcessStep has the following object properties:
```
described by sequenceNumber (note "Sequence identifier of processing steps in chronological order.") with a single value of type int,
described by personnel with values of type Person,
described by organization (note "The organization which performs the process step.") with a single value of type Organization,
described by address (note "Address where process was performed.") with a single value of type Address,
described by processStartTime (note "Time when process step starts.") with a single value of type dateTime,
described by processEndTime (note "Time when process step ends.") with a single value of type dateTime,
described by processStepSpecification note("Prescribed process specification for this process occurrence.") with a single value of type ProcessSpecification,
described by processStepData (note "Process monitoring data, in-situ monitoring data, real-time monitoring etc, real-time control information.") with a single value of type ProcessStepD
described by name (note 'Human readable name which designates this process step.') with a single value of type string,
described by processStepSpecification with a single value of type ManufacturingSpecification,
described by processMachine (note "The machine used to transform the input material.") with a single value of type System,
described by controlSoftware (note "Software used to control the process.") with values of type Software,
described by consumables (note "Material which is used during the execution of a process, but does not, in a substantial way, become part of material output. Example: Process Gas") with values of type MaterialStock,
described by materialInputs (note "Material that is an input to the process and because a part of the material output in a substantial way. Examples: Powder Feedstock, billet, sub-components to be assembled.") with values of type MaterialStock,
described by processSimulation (note "Reference to Simulation records associated with this processStep.") with values of type ProcessSimulation.
```

Reply:
Related to https://github.com/kaggour/AM-CDM/pull/15#discussion_r1840347086 which shows the note for ManufacturingProcessStep would need to be generalized as well